Kitodo-API/src/main/java/org/kitodo/config/Config.java

```diff
@@ -11,141 +11,197 @@
 
 package org.kitodo.config;
 
+import java.net.URI;
+import java.nio.file.Paths;
 import java.util.NoSuchElementException;
+import java.util.Optional;
 
+import org.apache.commons.configuration.AbstractConfiguration;
 import org.apache.commons.configuration.ConfigurationException;
 import org.apache.commons.configuration.ConversionException;
 import org.apache.commons.configuration.PropertiesConfiguration;
 import org.apache.commons.configuration.reloading.FileChangedReloadingStrategy;
+import org.apache.commons.io.FilenameUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 public class Config {
     private static final Logger logger = LogManager.getLogger(Config.class);
     private static volatile PropertiesConfiguration config;
     private static final String CONFIG_FILE = "kitodo_config.properties";
-    private static final String METADATA_DIRECTORY = "MetadatenVerzeichnis";
-    private static final String CONFIG_DIR = "KonfigurationVerzeichnis";
```

Comment:
Naming of variables: 
- xxx_DIRECTORY
- xxx_DIR
- DIR_xxx

Probably it would be advisable to choose one naming convention.

Reply:
Strange. Seems to be you are seeing not the last commit here? This has been changed already.

Reply:
Strange. Seems to be you are seeing not the last commit here? This has been changed already.

Reply:
You commented on the removed lines. This has been fixed (all was changed to DIR_xxx).